### DISCUSSION TITLE
Need help regarding Mesh refinement and y+

### DISCUSSION THREAD

**Discussion by usmanshahid90** [mdolab/dafoam#140] [General]
Dear all, 
I am validating NACA0012 CFD by using the default mesh settings available for NACA0012 transonic case. I m running dafoam with only 1 design cycle.

The problem is, the results are deviating from the experimental results.  The reason might be the mesh size and y+ value.
I need to validate NACA 0012 cfd with high mesh quality but I am unable to refine mesh and run it in dafoam.
If anyone could make mesh settings with refined values and could share with me the mesh script, i would be really thankful. Y+  less than 1 is desireable and i also want to   run the whole optimization cycles.
Need it on urgent basis because of supervisor pressure. 

Regards, 
Usman Shahid

**Comment by usmanshahid90** [comment]
![16228345622232476286753826319956](https://user-images.githubusercontent.com/75417534/120852764-126e9200-c594-11eb-9904-fb41cfaf9138.jpg)
These are my results

**Reply by friedenhe** [reply]
Check [here](https://dafoam.github.io/mydoc_get_started_faq.html#how-to-use-a-finer-mesh) on how to refine the mesh. If you are interested in knowing more details, sign up for the online workshop from [here](https://dafoam.github.io/dafoam_workshop_2021.html).
